lib/ui/theme/theme.dart

```diff
@@ -20,6 +21,7 @@ class PostsTheme {
           ),
         ),
         iconTheme: IconThemeData(color: colorScheme.primary),
+        brightness: brightnessTheme,
```

Comment:
Why have you added this?

Reply:
Fixes the white appbar color

![106564211_559877284687760_420802576461646916_n](https://user-images.githubusercontent.com/42913823/86695749-677ded00-c03f-11ea-981c-4a1da7735d16.jpg)